src/main/java/slimeknights/tconstruct/blocks/WorldBlocks.java

```diff
@@ -0,0 +1,217 @@
+package slimeknights.tconstruct.blocks;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.registries.ObjectHolder;
+
+import slimeknights.tconstruct.TConstruct;
+import slimeknights.tconstruct.common.registry.BaseRegistryAdapter;
+import slimeknights.tconstruct.common.registry.ItemRegistryAdapter;
+import slimeknights.tconstruct.library.TinkerRegistry;
+import slimeknights.tconstruct.shared.block.CongealedSlimeBlock;
+import slimeknights.tconstruct.shared.block.OreBlock;
+import slimeknights.tconstruct.shared.block.SlimeBlock;
+import slimeknights.tconstruct.world.block.SlimeDirtBlock;
+import slimeknights.tconstruct.world.block.SlimeGrassBlock;
+import slimeknights.tconstruct.world.block.SlimeLeavesBlock;
+import slimeknights.tconstruct.world.block.SlimeSaplingBlock;
+import slimeknights.tconstruct.world.block.SlimeTallGrassBlock;
+import slimeknights.tconstruct.world.block.SlimeVineBlock;
+import slimeknights.tconstruct.world.worldgen.BlueSlimeTree;
+import slimeknights.tconstruct.world.worldgen.MagmaSlimeTree;
+import slimeknights.tconstruct.world.worldgen.PurpleSlimeTree;
+
+import static slimeknights.tconstruct.common.TinkerPulse.injected;
+
+@ObjectHolder(TConstruct.modID)
+public final class WorldBlocks {
+
+  /* Ores */
+  public static final OreBlock cobalt_ore = injected();
+  public static final OreBlock ardite_ore = injected();
+
+  /* Slimestuff */
+  public static final SlimeBlock blue_slime = injected();
+  public static final SlimeBlock purple_slime = injected();
+  public static final SlimeBlock blood_slime = injected();
+  public static final SlimeBlock magma_slime = injected();
+  public static final SlimeBlock pink_slime = injected();
+
+  public static final CongealedSlimeBlock congealed_green_slime = injected();
+  public static final CongealedSlimeBlock congealed_blue_slime = injected();
+  public static final CongealedSlimeBlock congealed_purple_slime = injected();
+  public static final CongealedSlimeBlock congealed_blood_slime = injected();
+  public static final CongealedSlimeBlock congealed_magma_slime = injected();
+  public static final CongealedSlimeBlock congealed_pink_slime = injected();
```

Comment:
We also use congealed slime for gadgets in the slime channel recipe, would it make sense to move this to common?

You could argue the same for the slime blocks as they would be expected even apart from the world module.

Reply:
I classified the things by "is in the world/generated" as well as how they show up in the creative tabs. Might move them